venusian/tests/test_venusian.py

```diff
@@ -191,6 +191,21 @@ def test_classdecorator(self): # pragma: no cover
                              classdecorator.SuperClass)
             self.assertEqual(test.registrations[1]['superclass'], True)
 
+        def test_class_and_method_decorator(self):
+            from venusian.tests.fixtures import class_and_method
+            test = Test()
+            scanner = self._makeOne(test=test)
+            scanner.scan(class_and_method)
+            self.assertEqual(len(test.registrations), 2)
+            self.assertEqual(test.registrations[0]['name'], 'Class')
```

Comment:
I think @pieterproigia is correct, it seems consistent with the [docs](http://docs.pylonsproject.org/projects/venusian/en/latest/#using-venusian):

    name
        This is the module-level name of the object being decorated.
    ob
        This is the object being decorated if it’s a function or an instance; if the object being decorated is a method, however, this value will be the class.

To me that reads that `name` will be the name of `ob`.

I can also confirm that the fix works for me.